contracts/protocol/tokenization/NTokenApeStaking.sol

```diff
@@ -81,19 +96,88 @@ abstract contract NTokenApeStaking is NToken {
         address receiverOfUnderlying,
         uint256[] calldata tokenIds
     ) external virtual override onlyPool nonReentrant returns (uint64, uint64) {
-        ApeStakingLogic.executeAutoWithdraw(
-            _ERC721Data.owners,
-            _apeCoinStaking,
-            POOL_ID(),
-            tokenIds,
-            from
+        require(
```

Comment:
same here. we are pretty much preventing liquidation instead of unstaking which can be risky for our protocol